src/utils/queryHelpers.js

```diff
@@ -0,0 +1,29 @@
+import { computed, unref } from 'vue'
+import { isNetworkError, isServerError, isValidationError } from '@/utils/datastore/helpers'
+
+export function extractCursor (url) {
+  if (!url) return null
+  return new URL(url, url.startsWith('http') ? null : 'https://karrot.world').searchParams.get('cursor')
```

Comment:
That domain probably shouldn't be hardcoded here...

Not sure I understand the reason for this, is it related to local dev?

Reply:
This function is to take something like: `/api/offers/?status=active&group=12&cursor=abc123`

and turn it into `abc123`

So, just extracting the query parameter from a path. The domain there is just because the URL constructor requires a domain.

I agree it looks confusing though, an improvement would be to just initialize a `URLSearchParams` object directly (and use substring on the `?` character), I'll do that.